File: app/src/main/java/com/example/airmonitorble/SensorReading.kt
package com.example.airmonitorble

import com.google.gson.annotations.SerializedName

data class SensorReading(
    @SerializedName("timestampLong") val Timestamp: Long,
    @SerializedName("lpg_ppm") val Lpg: Double,
    @SerializedName("co2_ppm") val Co2: Double,
    @SerializedName("nh3_ppm") val Nh3: Double,
    @SerializedName("temp_c") val Temperature: Double,
    @SerializedName("humidity") val Humidity: Double,
    @SerializedName("aqi") val Aqi: Double
)

File: APIGas/Models/SensorReading.cs
namespace APIGas.Models
{
    public class SensorReading
    {
        public string Timestamp { get; set; }  // stored as string in Firebase
        public long TimestampLong { get; set; } // numeric (for sorting/graph)
        public string ReadableTimestamp { get; set; } 
        public double Temp_c { get; set; }
        public double Humidity { get; set; }
        public double Lpg_ppm { get; set; }
        public double Co2_ppm { get; set; }
        public double Nh3_ppm { get; set; }
        public double Aqi { get; set; }
    }
}
